sdks/python/apache_beam/utils/proto_utils.py

```diff
@@ -40,13 +40,13 @@
 @overload
 def pack_Any(msg):
   # type: (message.Message) -> any_pb2.Any
-  pass
+  ...
```

Comment:
pass vs ... are similar but not same. pass has an inferred return type of None for some type checkers. Pyright will give you an error for using pass here. ... is typical way to mark overloads as stubs.

Reply:
Are test files even type checked? I tried running mypy on this file and got an error. Also discovered mypy.ini for the repo has an invalid value giving this error message,

`mypy.ini: [mypy]: follow_imports: invalid choice 'true' (choose from 'normal', 'silent', 'skip', 'error')`

The mypy error that prevents it from type checking this file is,

`apache_beam/portability/api/metrics_pb2.pyi:117: error: invalid syntax  [syntax]` Looks like root cause of that error is a comment using type: notation for doc string and confusing it with standard type comment.

Reply:
Interesting, I think generated _pb2.py files should be excluded from the type checks.

Reply:
Didn't know about this usage of Ellipsis. Found: https://mypy.readthedocs.io/en/stable/stubs.html and https://github.com/python/typing/issues/109 where this is disussed.

Reply:
The pb2.py is not type checked, but pb2.pyi is needed for type inference. Looks like proto file has a comment that confuses mypy. I can make a tweak to the comment to make it fine for mypy.